core/src/test/scala/unit/kafka/zk/EmbeddedZookeeper.scala

```diff
@@ -37,6 +37,8 @@ class EmbeddedZookeeper() {
   factory.startup(zookeeper)
   val port = zookeeper.getClientPort()
 
+  println("Starting embedded zookeeper on port " + port + " using directories " + snapshotDir + "," + logDir)
```

Comment:
How about creating a test branch with println and create builds for Jenkins testing queue?